boards/arm/efm32pg_stk3402a/doc/efm32wg_stk3800.rst

```diff
@@ -0,0 +1,168 @@
+.. _efm32pg_stk3402a:
+
+EFM32PG-STK3402A
+###############
+
+Overview
+********
+
+The EFM32 Pearl Gecko Starter Kit EFM32PG-STK3402A contains a MCU from the
+EFM32PG family built on ARM® Cortex®-M4F processor with excellent low
+power capabilities.
+
+Hardware
+********
+
+- Advanced Energy Monitoring provides real-time information about the energy
+  consumption of an application or prototype design.
+- Ultra low power 128x128 pixel Memory-LCD
+- 2 user buttons, 2 LEDs and a touch slider
+- Humidity, temperature, and inductive-capacitive metal sensor
+- On-board Segger J-Link USB debugger
+
+For more information about the EFM32PG SoC and EFM32PG-STK3402A board:
+
+- `EFM32PG Website`_
+- `EFM32PG Datasheet`_
+- `EFM32PG Reference Manual`_
+- `EFM32PG-STK3402A Website`_
+- `EFM32PG-STK3402A User Guide`_
+- `EFM32PG-STK3402A Schematics`_
+
+Supported Features
+==================
+
+The efm32pg_stk3402a board configuration supports the following hardware features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| SYSTICK   | on-chip    | systick                             |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| USART     | on-chip    | serial port-polling;                |
+|           |            | serial port-interrupt               |
++-----------+------------+-------------------------------------+
+
+The default configuration can be found in the defconfig file:
+
+	``boards/arm/efm32pg_stk3402a/efm32pg_stk3402a_defconfig``
+
+Other hardware features are currently not supported by the port.
+
+Connections and IOs
+===================
+
+The EFM32PG SoC has six gpio controllers (PORTA to PORTF), but only four are
+currently enabled (PORTA, PORTB, PORTD and PORTF) for the EFM32PG-STK3402A board.
+
+In the following table, the column Name contains Pin names. For example, PE2
```

Comment:
Make the column name bold by changing to ``**Name**``
Change ``Pin`` to ``pin`` here and on the next line.

Reply:
Done